alternator/server.cc

```diff
@@ -235,7 +235,7 @@ class options_handler : public gated_handler {
 future<std::string> server::verify_signature(const request& req, const chunked_content& content) {
     if (!_enforce_authorization) {
         slogger.debug("Skipping authorization");
-        return make_ready_future<std::string>("<unauthenticated request>");
+        return make_ready_future<std::string>();
```

Comment:
Maybe we should use std::optional<sstring> for the user?

Is it possible for someone to send an empty string for the user?

In any case, I think it's outside the scope of the series.

Reply:
I think it is fine for us not to allow an empty-string user name, and consider the empty user name as "not authenticated at all".